frontend/src/modules/editor/selectTheme/less/editorTheme.less

```diff
@@ -20,33 +20,51 @@
   transition:all 0.3s;
   border: 1px solid #d6f2f9;
 
+  &-inner {
+    .theme-header {
+      padding: 20px 0;
```

Comment:
Can the padding be made equal so we avoid text hitting the side of the container? We then won't need it declared for `.display-name` or `.description`.

![padding](https://user-images.githubusercontent.com/922987/29716399-2f892f16-89a3-11e7-8542-77d7c24ce38c.png)